CONTRIBUTING.md

```diff
@@ -35,6 +35,45 @@ sure your PR updates all of:
 1. the `Package-Requires:` line in `racket-mode.el`
 2. the `deps` target in `makefile`
 
+## Pointing Emacs to your Git checkout
+
+After ensuring all dependencies of Racket More are installed, it
+suffices to add the path to your local checkout of Racket Mode to
+`load-path` and require the package:
+
+```elisp
+(add-to-list 'load-path "/path/to/the/git-clone/dir")
+(require 'racket-mode)
+```
+
+Note that these lines will override any previous references to Racket
+Mode in your Emacs configuration.  In particular, if you have Racket
+Mode installed as an Emacs package, after evaluating these lines you
+will use Racket Mode from your Git checkout.
+
+### Using `use-package`
+
+If you use `use-package`, you can simply replace
+
+```elisp
+(use-package racket-mode
+    :ensure t)
+```
+
+with
+
+```elisp
+(use-package racket-mode
+    :load-path "/path/to/the/git-clone/dir")
+```
+
+You might also need to:
```

Comment:
This advice is supposed to apply to both methods -- using `(add-to-path 'load-path _)` or `(use-packge :load-path __)`. But with the big "Using use-package" heading, it feels like you're saying it only applies to that method?  Maybe just delete that heading?

Reply:
By "this advice" I meant lines 70-76 -- which is advice I'd originally given you. I'm not even sure if it's strictly necessary. Sometimes Emacs can feel like a big pile of mutable state, which I why I advise deleting the MELPA package and restarting Emacs, just to be sure.

Anyway, the advice was meant for both ways of changing the load-path -- either directly or via use-package.